assets/js/linear_algebra/dot_product_formula2d.js

```diff
@@ -0,0 +1,273 @@
+var dot_product_formula2d = (function() {
+
+var origin = [150, 140], 
+  scale = 60, 
+  scatter = [], 
+  axis = [],
+  expectedAxis = [],
+  startAngleX = Math.PI,
+  startAngleY = 0.,
+  startAngleZ = 0.;
+let  axis_len = 2;
+
+let unit = axis_len/10;
```

Comment:
merge these two lines into the same "let"?

Reply:
done